articles/openshift/howto-deploy-java-eap-app.md

```diff
@@ -0,0 +1,384 @@
+
+1. Finally, click on the `Install` button to start the installation. This will open the `Topology view` with a graphical representation of the Helm release (named eap-todo-list-demo) and its associated resources.
+
+    ![OpenShift console topology](./media/howto-deploy-java-eap-app/console_topology.png)
+
+    The Helm Release (abbreviated HR) is named `eap-todo-list-demo`. It includes a Deployment resource (abbreviated D) also named `eap-todo-list-demo`. 
+
+1. When build is finished (the bottom-left icon will display a green check) and the application is deployed (the circle outline is in dark blue), you can go to application the URL (using the top-right icon) from the route associated to the deployment.
+
+    ![OpenShift console open application](./media/howto-deploy-java-eap-app/console_open_application.png)
+
+1. The application is opened in your browser looking similar to the following image:
+
+    ![OpenShift application running](./media/howto-deploy-java-eap-app/application_running_openshift.png)
+
+1. The application shows you the name of the pod which has served the information. To verify the clustering capabilities you could add some Todos, change the filter to see all the records and mark some of them as completed. Then delete the pod name that appears on the application, and once deleted, create a new Todo on the same application window. You will see that the new Todo is added via an ajax request, and you will continue getting the same list. It means your view, which was stored on the HTTP Session, has not been reset to a new one when the response is served by a different pod, indeed you will see that the session ID has not changed. However, this time the response was served by a different pod. That verifies that your HTTP Session data is being replicated across all your pods.
+
+## Clean up resources
+
+### Delete the application
+
+If you only want to delete your application, you can open the OpenShift console and, at the developer view, navigate to the `Helm` menu option. On this menu, you will see all the Helm Chart releases installed on your cluster. 
+
+   ![OpenShift uninstall application](./media/howto-deploy-java-eap-app/console_uninstall_application.png)
+
+Locate the `eap-todo-list-demo` Helm Chart and at the end of the row, click on the tree dots to open the action contextual menu entry.
+
+Click on "Uninstal Helm Release" to remove the application. Notice the application config map is not part of the chart and you need to remove it separately if you no longer need it.
+
+Execute the following command if you want to delete the configmap that hold the application configuration:
+
+```bash
+todo-list (bootable-jar-openshift) $  oc delete cm/todo-list-cm
+configmap "todo-list-cm" deleted
+```
+
+### Delete the database
+
+If you only want to delete the database and the related objects, execute the following command:
```

Comment:
Nit: "only" in this sentence implies you'd perhaps drop the db but leave the app, which sounds odd.

Reply:
Yes, right, although if you are testing things, maybe you want to know how to delete the database resources. In any case, yes, sounds a bit odd, so I removed the "only"